xdsl/dialects/arith.py

```diff
@@ -418,6 +418,43 @@ def __init__(
             attributes={"predicate": IntegerAttr.from_int_and_width(arg, 64)},
         )
 
+    @classmethod
+    def parse(cls, parser: Parser):
+        arg = parser.parse_identifier()
+        parser.parse_punctuation(",")
+        operand1 = parser.parse_unresolved_operand()
+        parser.parse_punctuation(",")
+        operand2 = parser.parse_unresolved_operand()
+        parser.parse_punctuation(":")
+        input_type = parser.parse_type()
+        (operand1, operand2) = parser.resolve_operands(
+            [operand1, operand2], 2 * [input_type], parser.pos
+        )
+
+        return cls(operand1, operand2, arg)
+
+    def print(self, printer: Printer):
+        printer.print(" ")
+        cmpi_comparison_operations = [
```

Comment:
Can you please move this to `_CMPI_COMPARISON_OPERATIONS` either on the file or module level?

Reply:
put it by other file globals